spec/unit/type/mount_spec.rb

```diff
@@ -530,6 +530,11 @@ def run_in_catalog(*resources)
       run_in_catalog(resource)
     end
 
+    it "should detect out of sync options" do
+      #provider.property_hash[:live_options] = "foo";
```

Comment:
I'm not sure what to do about this test. It technically works (e.g. it fails without the fix and passes with it) but that is only because if there are no live options retrieved from the output of the mount command, they default to an empty string. I would feel better if we could actually stub that value to something, but whenever I try to add live_options to the property hash of the provider the test freaks out because there's an unrecognized parameter.